lib/msal-browser/package.json

```diff
@@ -57,6 +57,7 @@
         "@babel/preset-typescript": "^7.7.2",
         "@babel/register": "^7.7.0",
         "@istanbuljs/nyc-config-babel": "^2.1.1",
+        "@trust/webcrypto": "^0.9.2",
```

Comment:
I very much disagree with this. This would also require security review.

Reply:
This also appears to be a Node-only package.

Reply:
And there's a warning on their Github not to use it.

Reply:
https://github.com/anvilresearch/webcrypto

Reply:
This is only for testing

Reply:
I need an implementation of browser crypto for mocha tests. Please provide an alternative.

Reply:
Ah, just figured that out. Disregard for now.